packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -9,6 +9,7 @@ import 'package:flutter/material.dart' show Tooltip;
 import 'package:flutter/rendering.dart';
 import 'package:flutter/scheduler.dart';
 import 'package:flutter/services.dart';
+import 'package:leak_tracker_flutter_testing/leak_tracker_flutter_testing.dart';
```

Comment:
Would you have to update the pubspec.yaml to declare this as a public dependency?

Reply:
It is already here: https://github.com/flutter/flutter/pull/137069